en/frames_multicopter/matrice100.md

```diff
@@ -0,0 +1,69 @@
+# DJI Matrice 100 (Pixhawk 1)
+
+The DJI Matrice quadcopter is a fully customizable and programmable flight platform. This build log provides build and configuration instructions for using the frame with the *3DR Pixhawk* flight controller.
+
+Key information:
+
+- **Frame:** DJI Matrice 100
+- **Flight controller:** [Pixhawk 1](../flight_controller/pixhawk.md)
+
+![DJI Matrice 100](../../assets/airframes/multicopter/matrice100/Matrice100.jpg)
+
+## Parts List
+
+  * [DJI Matrice 100](http://store.dji.com/product/matrice-100) Just ESCs motors, and frame.
+
+## Motor Connections
+
+See the [Pixhawk Wiring Quickstart](../assembly/quick_start_pixhawk.md) for autopilot assembly instructions.
+
+### Wiring Diagram
+
+![Connections](../../assets/airframes/multicopter/matrice100/Wiring Diagram.jpg)
+
+### Wiring Harness 
+
+![Wiring Harness](../../assets/airframes/multicopter/matrice100/WiringHarness.jpg)
+
+### PWM Connections
+ 
+![PWM Connections](../../assets/airframes/multicopter/matrice100/PwmInput.jpg)
+
+### Other views
+
+![Top](../../assets/airframes/multicopter/matrice100/Top.jpg)
+
+![Back](../../assets/airframes/multicopter/matrice100/Back.jpg)
+
+![No Stack](../../assets/airframes/multicopter/matrice100/NoStack.jpg)
+
+![No Top Deck](../../assets/airframes/multicopter/matrice100/NoTopDeck.jpg)
+
+### Autopilot Outputs
+
```

Comment:
@jgoppert I'm migrating this doc to PX4 user guide from devguide. The outputs here do not match those in the airframe reference: https://docs.px4.io/en/airframes/airframe_reference.html##copter_quadrotor_x_dji_matrice_100

Should this be changed below?

(Note, I will merge this anyway, but fix as a post process)

Are there any other things that need to be updated in this document that you know of?